apps/cf-ai-backend/src/index.ts

```diff
@@ -65,14 +64,18 @@ app.post("/api/add", zValidator("json", vectorObj), async (c) => {
   const { store } = await initQuery(c);
 
   console.log(body.spaces);
-  await batchCreateChunksAndEmbeddings({
+  const chunks = chunkText(body.pageContent, 1536);
+  if (chunks.length > 20) {
+    return c.json({ status: "error", message: "no chunks over 20" });
```

Comment:
I assume this message is shown to the user, maybe we can keep the errors consistent for the user and just say "This document is too large" or something?

Reply:
How about I change this one to "We are unable to process documents this size just yet, try something smaller"?

Reply:
yes works!